app/navigation/RepresentativeNavigator.scala

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package navigation
+
+import com.google.inject.Singleton
+import models._
+import pages._
+import pages.representative._
+import play.api.mvc.Call
+
+import javax.inject.Inject
+
+@Singleton
+class RepresentativeNavigator @Inject() () extends Navigator {
+
+  override def normalRoutes(departureId: String, mode: Mode): PartialFunction[Page, UserAnswers => Option[Call]] = ???
```

Comment:
Does this need implementing?

Reply:
We don't need to implement that at the moment as we are pulling the answers initially from IE015/13, this way users don't have any interaction with Representative answers till the CYA, so we technically are just using the checkRoutes, I have just kept that line in case things change down the line. Happy to remove it though.